### PR TITLE
Remove ObjectMapper

### PR DESCRIPTION
**Pull request by benaadams**
Just an experiment to see if Interfaces can be used instead of object mapping.

Full model binding to interface can't be done as while MVC will instantiate from DI; it won't then write the properties from the input stream https://github.com/aspnet/Mvc/issues/6014#issuecomment-288471434 

Luckily there is no interface model binding on this occasion

e.g. `AdminModel : IAdminViewModel, IAdminInputModel`

So that's not a problem here.

### CODE REVIEW

**Comment by benaadams** [comment]
Don't know if it works; are no tests 😉

File: src/live.asp.net/Controllers/AdminController.cs [hunk 1]
```diff
@@ -89,12 +93,9 @@ public class AdminInputModel
                 input.LiveShowEmbedUrl = "https://" + input.LiveShowEmbedUrl.Substring("http://".Length);
             }
 
-            TrackShowEvent(input, liveShowDetails);
+            TrackShowEvent(input, input);
```
Comment (JimiC) [inline src/live.asp.net/Controllers/AdminController.cs:96]: Why the double `input`? From what I can figure out the result of line 79 should be passed as the second parameter.

Comment (benaadams) [reply src/live.asp.net/Controllers/AdminController.cs:96]: Second one is `ILiveShowDetails` don't think the logic in `TrackShowEvent` works for the new approach though 😢

Comment (JimiC) [reply src/live.asp.net/Controllers/AdminController.cs:96]: Exactly what I thought.

**Pull request closed without merging**